models/bridge/zksync/bridge_zksync_native_flows.sql

```diff
@@ -0,0 +1,174 @@
+{{ config(
+    schema = 'bridge_zksync_native',
+    alias = 'flows',
```

Comment:
not a huge deal, but if this eventually turns into a sector-level spell with other chains, we'll want to follow standards more:
```suggestion
    schema = 'bridge_zksync',
    alias = 'native_flows',
```

would this make sense?

Reply:
I did `zksync_native` so as to refer to this as the native zkSync Era bridge. For Synapse Protocol, I would do something like `bridge_synapse_protocol`.`flows`. I'm not set on this though, so let you know if you would still like it be something different.

The Optimism bridge spell uses `flows` as the alias, so it may make sense to keep that.

Reply:
i'm good as-is for now. we can revisit later if we end up standardizing bridge spells across spellbook. i know you had an open issue on that.